src/ast/ast_validations.c

```diff
@@ -1747,13 +1739,12 @@ static VISITOR_STRATEGY _Validate_RETURN_Clause
 
 	// introduce bound vars
 	for(uint i = 0; i < num_return_projections; i ++) {
-		const cypher_astnode_t *child = cypher_ast_return_get_projection(n, i);
-		const cypher_astnode_t *alias_node = cypher_ast_projection_get_alias(child);
-		if(alias_node == NULL) {
-			continue;
+		const cypher_astnode_t *proj = cypher_ast_return_get_projection(n, i);
+		const char *alias = AST_GetProjectionAlias(proj);
```

Comment:
In which case `AST_GetProjectionAlias` returns `NULL` ?

Reply:
It can no longer return NULL, the condition is no longer needed.

Reply:
OK. The condition was removed.